<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow
{
	background-color:rgb(33,33,33);
	background-image:url( H:/Github/PythonScripts/PyQtDesigner/Watcher/bg7.jpg);
}


QWidget#centralwidget
{

	/*background-image:url( H:/Github/PythonScripts/PyQtDesigner/Watcher/bg1.jpg);*/
	background-color:rgb(33,33,33,0.5);
}

QLabel#labelUid
{
	color:rgb(224, 224, 224);
}

QLabel#labelMotionLogs
{
	color:rgb(224, 224, 224);
}

QLabel#labelWebTitle
{
	color:rgb(224, 224, 224);
}

QPushButton#startButton
{
font-size: 16px;

background-color: rgb(52, 140, 11);
border-radius: 25px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }

QPushButton#startButton:hover:pressed 
{
border-style: outset;
border-width:1.2px;

	
 }


QPushButton#stopButton
{
font-size: 16px;

background-color: rgb(229, 27, 27);
border-radius: 25px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }

QPushButton#stopButton:hover:pressed 
{
color:rgb(224, 224, 224);	
border-style: outset;
border-width:1.2px;
 }

QPushButton#saveLogsButton[flat=&quot;false&quot;]
{
font-size: 16px;
background-color: orange;
border-radius: 25px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
}


QPushButton#saveLogsButton:hover:pressed 
{ 
border-style: outset;
border-width:1.2px;
}

QLCDNumber#logsCount
{
color:rgb(220, 0, 0);
}






	
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetLogs">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="logsCount">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelWebCamera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSaveLogs">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>520</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Save Logs</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWebTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Webcamera Feed :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotionLogs">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Motion Logs :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>120</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelUid">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please enter user id :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSaveUid">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1325</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionManage_Users"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_User"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_Users">
   <property name="text">
    <string>Add User</string>
   </property>
  </action>
  <action name="actionAdd_User">
   <property name="text">
    <string>Manage Users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
